### ISSUE TITLE
Reproducing results for MS-COCO

### ISSUE THREAD

**Issue by akshitac8** [voyazici/orderless-rnn-classification#1]
Hello, @voyazici Thank you for sharing the codebase. I have some issues regarding the reproducibility of MS-COCO results. When I ran the exact code I got 4% less for C-F1 then I checked the image size. In the paper, you have mentioned that you change the image size to 288 for MS-COCO. I wanted to confirm is that the only change needed for reproducing the results

**Comment by voyazici** [comment]
Hello,
I re-launched the code with the given parameters to reproduce results and I noticed that I have given a lower learning rate for the training of the encoder model. I modified some hyperparameters including the image size and the learning rate, changed the commands in the README and re-launched the training. I got 77.46% overall and 74% per-class f1 scores. Let me know if you cannot reproduce the results.
Regarding to your other issue, I also made the code to support the multi-gpu training.

Best
